app/models/user/notifications/mentor_finished_discussion_notification.rb

```diff
@@ -0,0 +1,24 @@
+class User
+  module Notifications
+    class MentorFinishedDiscussionNotification < Notification
+      params :discussion
+
+      def url = discussion.student_url
+      def image_type = :avatar
+      def image_url = mentor.avatar_url
+      def guard_params = "Discussion##{discussion.id}"
+
+      def i18n_params
+        {
+          mentor_name: mentor.handle,
+          track_title: solution.track.title,
+          exercise_title: solution.exercise.title
+        }
+      end
+
+      private
+      def mentor = discussion.mentor
+      def solution = discussion.solution
```

Comment:
```ruby
delegate :mentor, :solution, to: :discussion
```

Reply:
The reason why rubocop doesn't suggest this is because these two methods are private. I assume `delegate` creates public methods, which is why it wasn't suggested.